templates/analytics/stats.html

```diff
@@ -67,7 +67,8 @@ <h1>{{ _("Messages sent by client") }}</h1>
                 <div class="chart-container">
                     <h1>{{ _("Active users") }}</h1>
                     <div class="button-container">
-                        <div class="buttons button-active-users">
+                        <label>{{ _("Aggregation") }}</label>
```

Comment:
What's the thinking behind the addition of this label?

Reply:
Matching the other two vertical bar/line graphs (messages sent over time and messages read).

If it doesn't make sense, we could just move the buttons to the left without a label. I think it would be better to have them not sit on top of the end of the graph.

Reply:
How about just dropping the "Aggregation" label on all the charts? I don't think it's helping me understand what's going on.

Reply:
Moving the buttons to the left without a label works for me.